### PR TITLE
Added my blog and added my photo into the students section

### PR DESCRIPTION
**Pull request by supunTE**

### CODE REVIEW

**Comment by supunTE** [comment]
https://stedirisuriya.github.io/gci17.fossasia.org/
https://stedirisuriya.github.io/gci17.fossasia.org/students.html

File: _data/blogs.yml [hunk 1]
```diff
@@ -47,6 +47,11 @@
   title: Root to Open-Source with FOSSASIA
   img: rsz_gutter.jpg
   author_name: Prasenjit Gautam
+  
+- url: https://morethanworld64569539.wordpress.com/2017/12/06/google-code-in-and-open-source/
+  title: Google Code-In and Open Source
+  img: 8.jpg
```
Comment (anshumanv) [inline _data/blogs.yml]: Please upload the image in the same PR.

**Comment by anshumanv** [comment]
Can you please remove commits which are not related to adding the blog, which is the purpose of this PR?

**Comment by STEdirisuriya** [comment]
https://stedirisuriya.github.io/gci17.fossasia.org/students.html

**Comment by supunTE** [comment]
This PR for add my blog and my photo for students section

**Comment by anshumanv** [comment]
Please try to achieve a single goal in one PR. Thanks

**Review by anshumanv — APPROVED** [review]

**Comment by supunTE** [comment]
https://stedirisuriya.github.io/gci17.fossasia.org/- see my blog
https://stedirisuriya.github.io/gci17.fossasia.org/students.html - see my photo on students page

**Pull request merged**